odoo/addons/base/tests/test_ir_actions_report.py

```diff
@@ -0,0 +1,102 @@
+from odoo import Command
+from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+from odoo.tests import tagged
+
+
+@tagged('post_install', '-at_install')
+class TestIrActionsReport(AccountTestInvoicingCommon):
+
+    def setup(self):
+        super().setup()
+        self.first_invoice = self.env['account.move'].create({
```

Comment:
`account` might not be installed

Reply:
So this test must be in `account`?

Reply:
Wouldn't it be possible to achieve the same goal using `base.report_irmodeloverview` or `base.report_irmodulereference`?

Reply:
Not sure. Now thinking about it, is there a specific reason for this to be independent of `account`?

Reply:
The bug was in `base`, not `account`

Reply:
Hmm, I've found [`test_reports.py`](https://github.com/odoo/odoo/blob/436be43d49260c905d37927b21b75404f4ccfc1f/odoo/addons/base/tests/test_reports.py) which exists in `base`. It tests base `ir_actions_report.py` functionality. Seems like the right place to put the tests.

Reply:
Looks like a good place indeed

Reply:
Hi. I've moved the tests into an appropriate place. Can you please take a look?